app/assets/javascripts/like.js

```diff
@@ -31,11 +31,17 @@ function clickliked() {
   // toggle liked to not liked.
   $.getJSON("/likes/node/" + node_id + "/delete")
    .done(function(response) {
-
-     shownotliked(node_id);
-     changelikecount(parseInt(response), node_id);
-     $('#like-button-' + node_id).on('click', clicknotliked);
-     $('#like-button-' + node_id).off('click', clickliked);
+    new Noty({
+      theme: 'mint',
```

Comment:
Could we reduce repetition by centralizing our default settings into `javascripts/noty.js` in a variable, and then this could become a single line?